src/main/java/blusunrize/immersiveengineering/common/crafting/ArcRecyclingRecipe.java

```diff
@@ -23,9 +24,10 @@
 {
 	private Map<ItemStack, Double> outputs;
 
-	public ArcRecyclingRecipe(Map<ItemStack, Double> outputs, Object input, int time, int energyPerTick)
+	public ArcRecyclingRecipe(Map<ItemStack, Double> outputs, IngredientWithSize input, int time, int energyPerTick)
 	{
-		super(null, input, ItemStack.EMPTY, time, energyPerTick);
+		//TODO: this needs an id
```

Comment:
How important is the ID (applies to the other `TODO`s as well)? And is arc recycling currently even implemented?